src/test/java/net/openhft/chronicle/wire/YamlWireTest.java

```diff
@@ -1365,24 +1358,24 @@ public void testStringMap() {
         @Nullable Map<String, String> map = wire.read().object(Map.class);
         assertEquals(0, map.size());
 
-        // TODO we shouldn't need to create a new wire.
-        // wire = createWire();
-//
-        // Set<String> threeObjects = new HashSet(Arrays.asList(new String[]{"abc", "def", "ghi"}));
-        // wire.write().object(threeObjects);
-//
-        // Set<String> list2 = wire.read()
-        // .object(Set.class);
-        // assertEquals(3, list2.size());
-        // assertEquals("[abc, def, ghi]", list2.toString());
+        // TODO we shouldn't need to create a new wire. should wire.reset()
```

Comment:
given this change, we should do this TODO

Reply:
I did, I need to fix the test now.

Reply:
I tried it locally and it worked